django/river/common/mapping/concept_maps.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+
+from common.adapters.fhir_api import HapiFhirAPI
+
+
+# FIXME: use this function on a mapping before sending it to the analyzer
+def dereference_concept_map(mapping, auth_token: Optional[str]):
+    for attribute in mapping["attributes"]:
+        for input_group in attribute["input_groups"]:
+            for input_ in input_group["inputs"]:
+                if concept_map_id := input_.get("concept_map_id"):
+                    concept_map = fetch_concept_map(concept_map_id, auth_token)
+                    input_["concept_map"] = concept_map
+
+
+def fetch_concept_map(concept_map_id: str, auth_token: Optional[str]):
```

Comment:
why desn't this method live in the `FhirAPI` adapter ?

Reply:
because it doesn't only fetch a concept map despite its name